memory/transparent_hugepages.py

```diff
@@ -91,6 +91,8 @@ def test(self):
         # Start Stresssing the  System
         self.log.info('Stress testing using dd command')
 
+        if not self.count:
+            self.fail("Please pass valid value for mem_size in yaml file")
```

Comment:
@Pavithra1602 where are we checking if `self.count >= 1G`
some time if `self.count < 1G` with some float value, then it should cancel the test right?

And please use `self.cancel` instead of `self.fail`

Reply:
@Pavithra1602 any update?

Reply:
@Naresh-ibm actually when 1gb hugepage is configured count value is coming as 0 when less than 2gb is configured in yaml file for mem_size
so added check to see if it is non zero

Reply:
> @Pavithra1602 where are we checking if `self.count >= 1G` some time if `self.count < 1G` with some float value, then it should cancel the test right?
> 
> And please use `self.cancel` instead of `self.fail`

@Pavithra1602 please address this